elm-app/src/elm/Events/Utils.elm

```diff
@@ -16,6 +16,16 @@ filterEvents events filterString =
         in
         DictList.filter
             (\_ event ->
+                let
+                    description =
```

Comment:
Minor, but as we seem to repeat this pattern you could also do something like

```elm
    let
        maybeStrings =
            [ Just event.name
            , event.description
            , Maybe.map (\location -> location.title) event.location
            ]
    in
    List.foldl
        (\maybeString accum ->
            unwrap accum
                (\string ->
                    if accum then
                        accum
                    else
                        stringMatch (String.toLower string)
                )
                maybeString
        )
        False
        maybeStrings
```

Reply:
😋